README.md

```diff
@@ -1,2 +1,25 @@
-# book-library
-Manchester Codes final project for Backend module to demonstrate knowledge of all topics covered
+# Book Library API Project
+---
```

Comment:
You dont need to add these lines in the readme, using the title blocks (the # values) will do this for you.

Also its general convention to only use one single # element for the title, a ## element for each section and a ### element for each subsection in these